instruments/thorlabs/thorlabsapt.py

```diff
@@ -218,7 +218,7 @@ def n_channels(self, nch):
         # If we add more channels, append them to the list,
         # If we remove channels, remove them from the end of the list.
         if nch > self._n_channels:
-            self._channel = self._channel + \
+            self._channel = list(self._channel) + \
```

Comment:
`self._channel` can be provided as tuple, but we clearly need a list here. This showed up in one of the tests now. So, let's make it a list.